server/backend/api/app/crud/crud_user_role.py

```diff
@@ -134,35 +137,44 @@ def create_user_role_assignment_many(
 
             # Check if child role exists or add a new child role
             child_role = find_or_create_forest_client_child_role(
-                db, forest_client_number, fam_role, requester
+                db, forest_client_number, fam_role, requester.cognito_user_id,
             )
             # Create user/role assignment
-            handle_create_return = create_user_role_assignment(
-                db, fam_user, child_role, requester
+            new_user_role_assginment_res = create_user_role_assignment(
+                db, fam_user, child_role, requester.cognito_user_id,
             )
-            create_return_list.append(handle_create_return)
+
+            # Update response object for Forest Client Name from the forest_client_search.
+            # FAM currently does not store forest client name for easy retrieval.
+            new_user_role_assginment_res.detail.role.forest_client = FamForestClientSchema.from_api_json(forest_client_search_return[0])
+            new_user_permission_granted_list.append(new_user_role_assginment_res)
     else:
         # Create user/role assignment
-        handle_create_return = create_user_role_assignment(
-            db, fam_user, fam_role, requester
+        new_user_role_assginment_res = create_user_role_assignment(
+            db, fam_user, fam_role, requester.cognito_user_id,
         )
-        create_return_list.append(handle_create_return)
+        new_user_permission_granted_list.append(new_user_role_assginment_res)
+    LOGGER.info(f"User/Role assignment executed successfully: {new_user_permission_granted_list}")
+
+    permissionAuditService = PermissionAuditService(db)
+    permissionAuditService.store_user_permissions_granted_audit_history(
+        requester, fam_user, new_user_permission_granted_list
+    )
 
-    LOGGER.debug(f"User/Role assignment executed successfully: {create_return_list}")
-    return create_return_list
+    return new_user_permission_granted_list
 
 
 def create_user_role_assignment(
-    db: Session, user: models.FamUser, role: models.FamRole, requester: str
+    db: Session, user: models.FamUser, role: models.FamRole, requester_cognito_user_id: str  #cognito_user_id,
```

Comment:
Remove the commented out code `#cognito_user_id`

Reply:
Ah~ thanks for catching that! Removed.